app-admin/etcd-wrapper/files/etcd-member.service

```diff
@@ -0,0 +1,26 @@
+[Unit]
+Description=etcd (System Application Container)
+Documentation=https://github.com/coreos/etcd
+Conflicts=etcd.service
+Conflicts=etcd2.service
+
+[Service]
+Type=notify
+Restart=always
+RestartSec=10s
+LimitNOFILE=40000
+TimeoutStartSec=60
+
+Environment="ETCD_IMG_TAG=v3.0.7"
```

Comment:
let's use 3.0.10. we can almost always try latest patch version of etcd. It is usually the most stable one. We probably want to hold off for upgrading to the next minor version for 1 or 2 patch release. The first release of a minor version bump is usually not stable.